cloudevents-sdk-rdkafka/src/headers.rs

```diff
@@ -0,0 +1,35 @@
+use cloudevents::event::SpecVersion;
+use lazy_static::lazy_static;
+use std::collections::HashMap;
+
+macro_rules! header_value_to_str {
+    ($header_value:expr) => {
+        str::from_utf8($header_value).map_err(|e| cloudevents::message::Error::Other {
+            source: Box::new(e),
+        })
+    };
+}
+
+macro_rules! attribute_name_to_header {
+    ($attribute:expr) => {
+        format!("{}{}", "ce_", $attribute)
+    };
+}
+
+fn attributes_to_headers(it: impl Iterator<Item = &'static str>) -> HashMap<&'static str, String> {
+    it.map(|s| {
+        if s == "datacontenttype" {
+            (s, String::from("content-type"))
+        } else {
+            (s, attribute_name_to_header!(s))
+        }
+    })
+    .collect()
+}
+
+lazy_static! {
+    pub(crate) static ref ATTRIBUTES_TO_HEADERS: HashMap<&'static str, String> =
+        attributes_to_headers(SpecVersion::all_attribute_names());
+    pub(crate) static ref SPEC_VERSION_HEADER: &'static str = "ce_specversion";
```

Comment:
for static strings you don't need the `lazy_static` macro, you can just use the static keyword https://doc.rust-lang.org/1.29.0/book/first-edition/const-and-static.html#static

Reply:
done!